.golangci.yml

```diff
@@ -1,54 +1,85 @@
+version: "2"
 run:
-  deadline: 5m
```

Comment:
No deadline in v2?

Reply:
It was never `deadline` - that did nothing. :grin:
Have introduced `timeout: 5m`.